cli/types/cypress.d.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types='chrome'/>
```

Comment:
Doesn't `@types/chrome` need to be in the dependency list for this to work? If that's the case, I think this might not be worth it to add for this one type.

Reply:
hmm good point